testcontainers/src/core/env/config.rs

```diff
@@ -126,11 +128,21 @@ impl Config {
     ///  2. `DOCKER_HOST` environment variable.
     ///  3. Docker host from the `docker.host` property in the `~/.testcontainers.properties` file.
     ///  4. Else, the default Docker socket will be returned.
-    pub(crate) fn docker_host(&self) -> &str {
+    pub(crate) fn docker_host(&self) -> Cow<'_, str> {
```

Comment:
we will also need to update documentation of the function and in `lib.rs`

Reply:
Done!